src/support_sphere/lib/presentation/pages/main_app/resource/resource_main_body.dart

```diff
@@ -26,9 +26,10 @@ class ResourceBody extends StatelessWidget {
     return BlocProvider(
       create: (context) => ResourceCubit(authUser),
       child: BlocBuilder<ResourceCubit, ResourceState>(
-        buildWhen: (previous, current) =>
-            previous.currentNav != current.currentNav ||
-            previous.resourceTypes != current.resourceTypes,
+        // buildWhen: (previous, current) =>
+        //     previous.currentNav != current.currentNav ||
+        //     previous.resourceTypes != current.resourceTypes ||
+        //     previous.resources != current.resources,
```

Comment:
Should we delete these lines? Or add a comment about when/if this should be enabled?

Reply:
Thanks for pointing this out! I forgot to delete it 😅